app/grandchallenge/components/models.py

```diff
@@ -158,9 +158,6 @@ class InterfaceSuperKindChoices(models.TextChoices):
 
 
 class InterfaceKind:
-    """Interface kind choices."""
-
-    InterfaceKindChoices = InterfaceKindChoices
```

Comment:
Are you sure that this is safe to remove? These things are often transported through for use in templates.